packages/common/src/allo/transaction-sender.ts

```diff
@@ -133,12 +142,18 @@ export function createViemTransactionSender(
         transactionHash: receipt.transactionHash,
         blockHash: receipt.blockHash,
         blockNumber: receipt.blockNumber,
-        logs: receipt.logs.map((log) => ({
+        logs: receipt.logs.map((log: Log) => ({
           data: log.data,
           topics: log.topics,
         })),
       };
     },
+
+    async address(): Promise<Address> {
+      const address = await walletClient?.account?.address;
+      if (!address) return "0x";
```

Comment:
Same here, if the address is empty then there could be something wrong, if this is expected then this function could just return `Address | null` and force the caller to handle it, otherwise this is silencing a potential issue.